# Normal Curve

The properties of the bell curve were studied by Carl Friedrich Gauss (1777 - 1855) and his contemporaries, and Pierre-Simon LaPlace (1749 - 1827) proved the first version of the Central Limit Theorem in 1810. The modern function that describes the Gaussian distribution or the **Normal Distribution** is as follows:

$$
f(x) = \frac{1}{\sigma \sqrt{2\pi} } e^{-\frac{1}{2}\left(\frac{x-\mu}{\sigma}\right)^2}
$$

where $\mu$ is the mean and $\sigma$ is the standard deviation. In this course, we use the notation $N(\mu, \sigma)$ to refer to a normal distribution with the indicated mean and standard deviation.

## Standard Normal

For the purposes of calcuation, especially when the integrals involved must be done by hand, mathematicians developed a simplification where

- $\mu = 0$
- $\sigma = 1$

One can readily verify by inspection of the above intgral that, computational, this simplication should make the integration more easily accomplished.

## Standardized Scores

Statistics before computers involved tables. Any score within the experimentally gathered data could be converted to a standardized score. In this way, the estimation or calculation of percentiles was possible for any data point in the sample data. A stadardized score called a $z$-score was computed, and the relevant value was found in a table.

The formula for calcuating a $z$-score is as follows:

$$z = \frac{x - \mu}{\sigma}$$

In modern statistical practice, we often estimate $\mu$ and $\sigma$. Thus, we often calculate $z$-scores based upon the sample average and sample standard deviation.

## Example 1

The ACT traditionally has scores that follow a $N(21,5)$ distribution. If Molly scored a 29 on the ACT, in what percentile was here score?

**$z$-score**

We first calculate her standardized score:

$$z_M = \frac{29 -21}{5} =\frac{8}{5} = 1.6$$

The percentile relative to this $z$-score can be found using the R command **pnorm()**:

In [16]:
pnorm(1.6)

[1] 0.9452007

Thus, Molly scored in the 94th percentile on the ACT.

## Example 2

What if we knew that Dolly scored in the 87th percentile on the ACT. What was her score?

First, we have to convert the percentile into a standardized score. For this, we use the R function **qnorm()**.

In [17]:
qnorm(0.8)

[1] 0.8416212

If $z \approx 0.8416$, then we can calcuate using the $z$-score formula:

$$0.8416 = \frac{ACT_D - 21}{5}$$

When we simplify and rearrange the equation, we find that:

$$ACT_D = 5 (0.8416) + 21$$

or equivalently that:

$$ACT_D \approx 25.21$$

Thus, Dolly scored a 25 on the ACT.